README.rst

```diff
@@ -105,6 +105,15 @@ Features
       +---------------------------+
       先手の持駒：　銀
 
+* Render board as SVG.
+
+  .. code:: python
+
+      >>> print(board.svg())
+      <svg>
```

Comment:
It would be great if we could embed a real image generated by the command `board.svg()`.
But I know it's not easy.